ConsistencyManager/ConsistencyManager.swift

```diff
@@ -388,7 +388,7 @@ public class ConsistencyManager {
     */
     public func cleanMemory() {
         dispatchTask { _ in
-            NSNotificationCenter.defaultCenter().postNotificationName(ConsistencyManager.kCleanMemoryAsynchronousWorkStarted, object: self)
+            NotificationCenter.default.post(name: Notification.Name(rawValue: ConsistencyManager.kCleanMemoryAsynchronousWorkStarted), object: self)
```

Comment:
This seems like the right way to do it. There could be an alternative where we initialize the `kCleanMemoryAsynchronousWorkStarted` variable of type `Notification.Name`. So the code will change to this:

```
public static let kCleanMemoryAsynchronousWorkStarted = Notification.Name("com.linkedin.consistencyManager.kCleanMemoryAsynchronousWorkStarted")
```

```
NotificationCenter.default.post(name: ConsistencyManager.kCleanMemoryAsynchronousWorkStarted, object: self)
```